### DISCUSSION TITLE
Text rendering problem on Farsi(fa) or Arabic (ar) locale

### DISCUSSION THREAD

**Discussion by alireza-es** [requarks/wiki#3496] [Error / Bug Report]
### Actual behaviour
I downloaded the "fa" locale and change my default language to "Farsi". 
### Expected behaviour
I expected to render pages normally, but most of the texts are like ????.
### Steps to reproduce the behaviour
1- Install wiki.js
2- Create the home page
3- Goto Settings
3- Download "fa"(فارسی) locale
4- Change the default language to fa(فارسی)

Environment:
- OS: Windows Server 2019
- Database: MSSQL Server 2017
- Database Collation: PERSIAN_100_CI_AS
- Wiki.js version: 1.2.159

![image](https://user-images.githubusercontent.com/3083080/95678070-ddfa3000-0bd6-11eb-8083-b9a9099caf2b.png)
![image](https://user-images.githubusercontent.com/3083080/95678096-35989b80-0bd7-11eb-8637-0f60063fdbfb.png)

**Comment by sadeq-qafari** [comment]
I checked the database and in mssql dbo.locales.strings column type is set to text witch does not support persian characters, problem will be fixed by setting the column type to nvarchar(max)

**Reply by niknam-mohsen** [reply]
I changed the dbo.locales.strings column type to nvarchar(max) and the problem of persian characters in the database was solved but in dashboard it still shows incorrect characters  like ????

**Reply by sadeq-qafari** [reply]
Its been a long time since I used wiki, I'm not sure how we managed to fix that! sorry.
